packages/extra/test/reference/DayOfYearTest.js

```diff
@@ -0,0 +1,524 @@
+/*
+ * @copyright (c) 2022, Philipp Thürwächter & Pattrick Hüper & Michał Sobkiewicz
+ * @copyright (c) 2007-present, Stephen Colebourne & Michael Nascimento Santos
+ * @license BSD-3-Clause (see LICENSE in the root directory of this source tree)
+ */
+
+import {
+    ChronoField,
+    ChronoUnit,
+    Clock,
+    DateTimeException,
+    IsoChronology,
+    IsoFields,
+    LocalDate,
+    LocalTime,
+    NullPointerException,
+    TemporalField,
+    TemporalQueries,
+    UnsupportedTemporalTypeException,
+    ValueRange,
+    Year,
+    ZoneId,
+} from '@js-joda/core';
+
+import '../_init';
+
+import { assertEquals, assertThrows } from '../testUtils';
+
+import { DayOfYear } from '../../src/DayOfYear';
+
+describe('org.threeten.extra.TestDayOfYear', () => {
+    const YEAR_STANDARD = Year.of(2007);
+    const YEAR_LEAP = Year.of(2008);
+    const STANDARD_YEAR_LENGTH = 365;
+    const LEAP_YEAR_LENGTH = 366;
+    const TEST = DayOfYear.of(12);
+
+    class TestingField extends TemporalField {
+        getBaseUnit() {
+            return ChronoUnit.DAYS;
+        }
+
+        getRangeUnit() {
+            return ChronoUnit.YEARS;
+        }
+
+        range() {
+            return ValueRange.of(1, 365, 366);
+        }
+
+        isDateBased() {
+            return true;
+        }
+
+        isTimeBased() {
+            return false;
+        }
+
+        isSupportedBy(temporal) {
+            return temporal.isSupported(ChronoField.DAY_OF_YEAR);
+        }
+
+        // eslint-disable-next-line no-unused-vars
+        rangeRefinedBy(temporal) {
+            return this.range();
+        }
+
+        getFrom(temporal) {
+            return temporal.getLong(ChronoField.DAY_OF_YEAR);
+        }
+
+        adjustInto(temporal, newValue) {
+            return temporal.with(ChronoField.DAY_OF_YEAR, newValue);
+        }
+    }
+
+    TestingField.INSTANCE = new TestingField();
+
+    //-----------------------------------------------------------------------
+    // now()
+    //-----------------------------------------------------------------------
+    describe('now()', () => {
+        it('test_now', () => {
+            let test = DayOfYear.now();
+            if (LocalDate.now().dayOfYear() !== test.value()) {
+                test = DayOfYear.now();
+            }
```

Comment:
same comment here

Reply:
Fixed in 878336ab4c1a6fe2cf892bc20a1725bae8628904.